src/js/components/Baani.js

```diff
@@ -116,6 +116,26 @@ export default class Baani extends React.PureComponent {
     }
   };
 
+  makeSelection = e => {
+    const shareDivs = document.querySelectorAll('.share');
+    shareDivs.forEach(function(sd) {
+      sd.style.display = 'none';
+    });
+    if (window.getSelection().toString()) {
+      const baaniDiv = e.currentTarget;
+      var range = document.createRange();
+      range.selectNode(baaniDiv);
+      window.getSelection().removeAllRanges();
+      window.getSelection().addRange(range);
+      const shareDiv = baaniDiv.querySelector('.share');
+      shareDiv.style.display = 'block';
+    }
```

Comment:
Let's make a helper function that takes a DOM node and selects it. That way the functionality and logic isn't coupled. Makes sense?

Reply:
Fixed.

Reply:
Let's keep this in helpers file so that other components can use as well. What do you think?

Reply:
Yeah, good idea. Done. :raised_hands: